analyzer/command/apk/apk_test.go

```diff
@@ -162,21 +307,17 @@ func TestAnalyze(t *testing.T) {
 	}
 	analyzer := alpineCmdAnalyzer{}
 	for testName, v := range tests {
-		f, err := os.Open(v.apkIndexArchivePath)
-		if err != nil {
-			t.Fatalf("unexpected error: %s", err)
-		}
-		apkIndexArchive := &apkIndex{}
-		if err = json.NewDecoder(f).Decode(apkIndexArchive); err != nil {
-			t.Fatalf("unexpected error: %s", err)
-		}
-		actual, _ := analyzer.Analyze(v.args.targetOS, v.args.configBlob)
-		sort.Slice(actual, func(i, j int) bool {
-			return actual[i].Name < actual[j].Name
+		t.Run(testName, func(t *testing.T) {
+			apkIndexArchiveURL = v.apkIndexArchivePath
+			actual, _ := analyzer.Analyze(v.args.targetOS, v.args.configBlob)
+			sort.Slice(actual, func(i, j int) bool {
+				return actual[i].Name < actual[j].Name
+			})
+			if !reflect.DeepEqual(v.expected, actual) {
```

Comment:
This is an old test. We recently use `testify/assert`. Let's replace it this time.

```suggestion
			assert.Equal(t, v.expected, actual)
```